test/functional/specs/C14407.js

```diff
@@ -35,22 +34,29 @@ const setConsentIn = ClientFunction(() => {
   return window.alloy("setConsent", { general: "in" });
 });
 
+const sendEventWithDelay = ClientFunction((data, delay) => {
```

Comment:
Should we make this a helper?  Also can you write a comment on why the delay is necessary and under what circumstances it should be used?